support-metrics-common/src/main/java/io/confluent/support/metrics/BaseMetricsReporter.java

```diff
@@ -245,7 +245,7 @@ protected void submitMetrics() {
       if (sendToKafkaEnabled() && encodedMetricsRecord != null) {
         // attempt to create the topic. If failures occur, try again in the next round, however
         // the current batch of metrics will be lost.
-        if (kafkaUtilities.createAndVerifyTopic(zkUtilsProvider().zkUtils(), supportTopic,
+        if (kafkaUtilities.createAndVerifyTopic(zkClientProvider().zkClient(), supportTopic,
```

Comment:
Close the zk client?

Reply:
Since we return the zk client from `KafkaServer`, we don't need to close it.